src/Tizen.NUI/src/public/Animation/TransitionOptions.cs

```diff
@@ -97,18 +107,36 @@ private void EnableProvider()
         }
 
         /// <summary>
-        /// Gets or sets the Shared object Id
+        /// During the Page transition, if two Views each of on the old top Page
+        /// and new top Page have same TransitionTag, the View on the old top Page
+        /// will be transition to the one of new top Page.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string TransitionTag
```

Comment:
If the property is just get/set the private field `transitionTag`, you can simplify by removing field and use [auto-implemented property](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/auto-implemented-properties)
```C#
public string TransitionTag { get; set; }
```

Reply:
thanks.